File: Desktop/GPS/app/src/main/java/com/example/jesus/gps/MainActivity.java
package com.example.jesus.gps;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {
    private Button Ubicame;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Ubicame  = (Button)findViewById(R.id.Ubicame);
        Ubicame.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent inten = new Intent(MainActivity.this,MapsActivity.class);
                startActivity(inten);
            }
        });
    }
}

File: AndroidStudioProjects/CasaDeCambio/app/src/main/java/com/example/xoana/casadecambio/MainActivity.kt
package com.example.xoana.casadecambio

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button
import android.widget.EditText

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        val pesos = findViewById<EditText>(R.id.pesostb) as EditText
        val euros = findViewById<EditText>(R.id.eurostb) as EditText
        val boton = findViewById<Button>(R.id.Convertir) as Button
      val con = pesos.text.toString()
        boton.setOnClickListener{
            euros.setText((pesos.text.toString().toDouble()*(0.04437)).toString())
        }
    }
}
